cmd/plan.go

```diff
@@ -68,7 +68,7 @@ func plan(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-func planInstall(rv cran.RVersion) (*cran.PkgNexus, gpsr.InstallPlan) {
+func planInstall(rv cran.RVersion, exitOnMissing bool) (*cran.PkgNexus, gpsr.InstallPlan) {
```

Comment:
should exitOnMissing be an argument here or a setting in the config file? Just some food for thought.

Reply:
no, its an internal knob that needs to be tuned for specific ways we want to use planInstall. Namely, this is really an artifact of the fact that plan install is probably too monolithic. We need some way of getting back a pkgNexus even if downstream solves fail, so this is the quick fix. 

We don't want the user mucking with this (eg config) because it could cause very un-intended behavior. Eg to install we _must_ have a full plan, so not existing would cause incorrect behavior (a blank install plan that might exit with a "success")